src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +17,15 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double AvgCurrent;
+    double inCurrent;
+    double changeTime;
+    QTime prevTime;
+    QTime currentTime;
+    double AmpHours;
+    double Current;
+    double AverageCurrent;
```

Comment:
What's the difference between `AverageCurrent` and `AvgCurrent`? Regardless, if they serve different purposes they should have different names.